test/integration/aens.ts

```diff
@@ -112,7 +113,7 @@ describe('Aens', function () {
   it('throws error on setting 33 pointers', async () => {
     const nameObject = await aeSdk.aensQuery(name)
     const pointers = Object.fromEntries(
-      new Array(33).fill().map((v, i) => [`pointer-${i}`, address])
+      new Array(33).fill(null).map((v, i) => [`pointer-${i}`, address])
     )
     expect(nameObject.update(pointers))
```

Comment:
```suggestion
    void expect(nameObject.update(pointers))
```

Otherwise throws `Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the 'void' operator.` error